### PR TITLE
Feat/support postgres password file

### PR DESCRIPTION
**Pull request by opichon**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -101,4 +102,4 @@ docker run \
 ## Database setup
 - Before you start this container, make sure to initialize your database.
 - You do not need to ensure that the Postgresql database has started before this container is started. During bootup, SuperTokens will wait for ~1 hour for a Postgresql instance to be available.
-- If `POSTGRESQL_USER`, `POSTGRESQL_PASSWORD` and `POSTGRESQL_CONNECTION_URI` are not provided, then SuperTokens will use an in memory database.
\ No newline at end of file
+- If `POSTGRESQL_USER`, `POSTGRESQL_PASSWORD` and `POSTGRESQL_CONNECTION_URI` are not provided, then SuperTokens will use an in memory database.
```
Comment (rishabhpoddar) [inline README.md:105]:
```suggestion
- If `POSTGRESQL_USER`, `POSTGRESQL_PASSWORD`, `POSTGRESQL_PASSWORD_FILE` and `POSTGRESQL_CONNECTION_URI` are not provided, then SuperTokens will use an in memory database.
```

File: docker-entrypoint.sh [hunk 2]
```diff
@@ -56,6 +56,11 @@ then
         echo "postgresql_user: $POSTGRESQL_USER" >> $CONFIG_FILE
     fi
 
+    if [ -n "${POSTGRESQL_PASSWORD_FILE}" ]; then
```
Comment (rishabhpoddar) [inline docker-entrypoint.sh:59]:
```suggestion
    if [ ! -z $POSTGRESQL_PASSWORD_FILE ]
    then
```

**Review by rishabhpoddar — CHANGES_REQUESTED** [review]

**Comment by rishabhpoddar** [comment]
hey @opichon thanks for this PR. Could you please have a look at some of my suggestions?

**Comment by opichon** [comment]
@rishabhpoddar Done

**Review by rishabhpoddar — APPROVED** [review]

**Comment by rishabhpoddar** [comment]
thanks @opichon. I will release this today.

I have also added you to the contributor's list in the supertokens-core repo :)

**Comment by rishabhpoddar** [comment]
@opichon released the image. The new image tag is the same as the older one as there is no change in the core - just an extra env

**Pull request merged**